scalameter-core/src/main/java/org/scalameter/execution/invocation/instrumentation/MethodInvocationCounter.java

```diff
@@ -0,0 +1,65 @@
+package org.scalameter.execution.invocation.instrumentation;
+
+import java.util.concurrent.atomic.AtomicLongArray;
+
+
+/**
```

Comment:
Please use the following syntax for ScalaDocs:

    /** First line that shortly describes the class.
     *  
     *  More lines that further clarify the meaning of the class.
     *  
     *  '''Note:''' special remarks are highlighted like this.
     */
    public class MethodInvocationCounter {

You should avoid using capital letters inside ScalaDoc comments, as they are not rendered nicely in online docs.

Please apply these guidelines to all your comments.